docs/DeveloperGuide.adoc

```diff
@@ -804,29 +817,51 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |new user |have commands that are intuitive and easy to remember |
+
+|`* * *` |user who perfers typing over using the mouse |be able to control almost everything from the CLI |
+
+|`* * *` |experienced user |have shortcut keys| do the same thing in a shorter time |
+
+|`* * *` |pair of NUS students in a relationship |view my partner's NUS timetable for the week |keep track of my partner's free time
+
+|`* * *` |pair of NUS students in a relationship |replace my partner's NUS timetable |see the most updated timetable
+
+|`* * *` |person in a relationship |add and view special events (e.g. Valentine's day) in the same month |better plan for these dates
+
+|`* * *` |person in a relationship |schedule meetings with my partner |
+
+|`* * *` |person in a relationship |add/view Journal entries |remember special memories
+
+|`* * *` |pair of NUS students in a relationship |locate my partner from his NUS timetable |easily meet up with my partner on campus
+
+|`* * *` |user who values their privacy |encrypt my NUSCouples save data |prevent strangers from reading personal information such as my journal entries
 
 |`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
 
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
```

Comment:
Will there be many persons? If not, then you need to remove unused/unrelated user stories, also in other sections.